README.md

```diff
@@ -239,6 +239,9 @@ amadeus.reference_data.locations.points_of_interest.get(latitude: 41.397158, lon
 # What are the popular places in Barcelona? (based on a square)
 amadeus.reference_data.locations.points_of_interest.by_square.get(north: 41.397158, west: 2.160873, south: 41.394582, east: 2.177181)
 
+# Airport On-Time Performance
+amadeus.airport.predictions.on_time.get(airportCode: 'JFK', date: '2020-03-01')
```

Comment:
I would propose to add a date even further in the future as this one is going to pass soon. In Python's readme we have `amadeus.airport.predictions.on_time.get(airportCode='JFK', date='2020-09-01')`.